### DISCUSSION TITLE
Can reacnetgenerator use in reaction on surface?

### DISCUSSION THREAD

**Discussion by jwz360** [deepmodeling/reacnetgenerator#909] [Q&A]
The surface is fixed. Thanks!

**Comment by njzjz** [comment]
It will work well if you give a trajectory that doesn't contain surface atoms (I think lammps has this feature). Or you can write a script to remove these atoms manually .

There is currently no such feature in reacnetgenerator to filter atom types.

**Comment by jwz360** [comment]
Thanks! It may be work. I will try it later.
